--- HypergeometricPolynomialTestTestSummand.text.nogood	2024-03-18 19:37:53.523739785 -0500
+++ HypergeometricPolynomialTestTestSummand.text.good	2024-03-18 19:37:23.851964574 -0500
@@ -107,7 +107,7 @@
     INVOKEVIRTUAL arb/Integer.set (Larb/Integer;)Larb/Integer;
     POP
    L2
-   FRAME SAME
+   FRAME SAME1 arb/Real
     ALOAD 0
     GETFIELD nzⁿTimesProdkαSubknWherekEquals1TopOvernBangTimesProdkβSubknWherekEquals1Toq.prodℝ1 : Larb/Real;
     ALOAD 0
@@ -163,7 +163,7 @@
     INVOKEVIRTUAL arb/Integer.set (Larb/Integer;)Larb/Integer;
     POP
    L3
-   FRAME SAME1 arb/Real
+   FRAME FULL [nzⁿTimesProdkαSubknWherekEquals1TopOvernBangTimesProdkβSubknWherekEquals1Toq java/lang/Object I I java/lang/Object] [arb/Real arb/Integer]
     ALOAD 0
     GETFIELD nzⁿTimesProdkαSubknWherekEquals1TopOvernBangTimesProdkβSubknWherekEquals1Toq.prodℝ2 : Larb/Real;
     ALOAD 0
@@ -202,8 +202,8 @@
     LOCALVARIABLE order I L0 L4 2
     LOCALVARIABLE bits I L0 L4 3
     LOCALVARIABLE result Larb/Real; L0 L4 4
-    MAXSTACK = 10
-    MAXLOCALS = 10
+    MAXSTACK = 7
+    MAXLOCALS = 5
 
   // access flags 0x1
   public <init>()V
@@ -281,8 +281,8 @@
     INVOKESPECIAL arb/Real.<init> ()V
     PUTFIELD nzⁿTimesProdkαSubknWherekEquals1TopOvernBangTimesProdkβSubknWherekEquals1Toq.ℝ3 : Larb/Real;
     RETURN
-    MAXSTACK = 10
-    MAXLOCALS = 10
+    MAXSTACK = 4
+    MAXLOCALS = 1
 
   // access flags 0x1
   public initialize()V
@@ -382,8 +382,8 @@
     ICONST_1
     PUTFIELD nzⁿTimesProdkαSubknWherekEquals1TopOvernBangTimesProdkβSubknWherekEquals1Toq.isInitialized : Z
     RETURN
-    MAXSTACK = 10
-    MAXLOCALS = 10
+    MAXSTACK = 4
+    MAXLOCALS = 1
 
   // access flags 0x1
   public close()V
@@ -424,6 +424,6 @@
     GETFIELD nzⁿTimesProdkαSubknWherekEquals1TopOvernBangTimesProdkβSubknWherekEquals1Toq.ℝ3 : Larb/Real;
     INVOKEVIRTUAL arb/Real.close ()V
     RETURN
-    MAXSTACK = 10
-    MAXLOCALS = 10
+    MAXSTACK = 1
+    MAXLOCALS = 1
 }
